opentelemetry-jaeger/README.md

```diff
@@ -109,15 +109,23 @@ example expects a Jaeger collector running on `http://localhost:14268`.
 
 ```toml
 [dependencies]
-opentelemetry-jaeger = { version = "..", features = ["collector_client"] }
+opentelemetry-jaeger = { version = "..", features = ["collector_client", "isahc"] }
```

Comment:
This should probably show the version that doesn't require manually setting the client

```suggestion
opentelemetry-jaeger = { version = "..", features = ["isahc_collector_client"] }
```